tests/contrib/pyodbc/test_pyodbc.py

```diff
@@ -282,6 +282,8 @@ def test_patch_unpatch(self):
             assert span.service == "pyodbc"
             assert span.name == "pyodbc.query"
             assert span.span_type == "sql"
+            assert span.get_tag("db.system") == "pyodbc"
```

Comment:
The value for "db.system" span, must be an identifier for the DBMS. For example, "mssql"?
**DD doc**: https://datadoghq.atlassian.net/wiki/spaces/APM/pages/2357395856/Span+attributes#db.system
**Otel doc**: https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/semantic_conventions/database.md